### PR TITLE
Dedicatoria, folha aprovacao, resumo and layout updated

### PR DESCRIPTION
**Pull request by TheVino**
Henrique, Lucas...

I am honestly asking you guys to view my changes.

### CODE REVIEW

**Comment by TheVino** [comment]
Updates: References, Abstract, Fundamentação, Introdução, layout, Capa [\addbibresource{References.bib}]

File: layout.tex [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 \author{
 	Henrique Casares\\
-	Vini Lopes\\
+	Vinicius Lopes Silva\\
 	Lucas Campos Martins Ferreira Braga
 }
 \orientador{Prof. Dr. Marco Antônio Assis de Melo}
```
Comment (LucasCMFBraga) [inline layout.tex]: Is it necessary change the name advisor ?

Comment (TheVino) [reply layout.tex]: We don't know yet all advisors, but we can use as _'placeholder'_

Comment (LucasCMFBraga) [reply layout.tex]: I thought our advisor was Batatão

**Review by LucasCMFBraga — APPROVED** [review]

**Pull request merged**